extensions/amp-mustache/amp-mustache.md

```diff
@@ -64,21 +64,31 @@ First, the `amp-mustache` has to be declared/loaded like this:
 <script async custom-template="amp-mustache" src="https://cdn.ampproject.org/v0/amp-mustache-0.2.js"></script>
 ```
 
-Then, the Mustache templates can be defined in the `template` tags like this:
+Then, the Mustache templates can be defined either in a `script` or `template` tag like this:
+
 
 ```html
+<!-- Using template tag. -->
 <template type="amp-mustache">
   Hello {{world}}!
 </template>
 ```
+or
 
-How templates are discovered, when they are rendered, how data is provided is  all decided by the
-target AMP element that uses this template to render its content (for example, in an [amp-list](../amp-list/amp-list.md), [amp-form](../amp-form/amp-form.md), etc.).
+<!-- Using script tag. -->
+```html
+<script type="text/plain" template="amp-mustache">
+  Hello {{world}}!
+</script>
+```
+
+Choosing one over the other is a matter of preference and needs. The script template solves an issue with templating in the context of tables; an issue when using template tags. See the "Tables" section further below.
```

Comment:
We should recommend using `<template>` wherever possible since AMP validation provides useful dev-x hints. `<script>` is a workaround for certain edge cases.